hotspot/src/share/vm/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -77,53 +81,19 @@
 // pointer is stored as integer value.
 #define NULL_WORD NULL
 
-// Some MS Visual Studio versions do not seem to have INT64_C and UINT64_C
-// even with __STDC_CONSTANT_MACROS defined.
-#ifndef INT64_C
-#define INT64_C(c)  (c ## i64)
-#endif
-#ifndef UINT64_C
-#define UINT64_C(c) (c ## ui64)
-#endif
```

Comment:
These removals aren't in the original changeset. See JDK-8272714 and JDK-8272214. I guess we should conditionalize on the VS version in use if that's a problem?